### PR TITLE
docs: add compare with parcel section

### PR DESCRIPTION
**Pull request by hardfist**
## Summary
add comparison with parcel section in introduction

## Checklist

- [x] Tests updated (or not required).
- [x] Documentation updated (or not required).

### CODE REVIEW

File: website/docs/zh/guide/start/introduction.mdx [hunk 1]
```diff
@@ -125,6 +125,10 @@ Rspack 和 turbopack 都是基于 Rust 实现的 bundler，且都发挥了 Rust
 
 Rspack 和 Rollup 虽然都是打包工具，但是两者的侧重领域不同，Rollup 更适合用于打包库，而 Rspack 适合用于打包应用。因此 Rspack 对打包应用进行了很多优化，如 HMR、Bundle splitting 等功能，而 Rollup 则比 Rspack 的编译产物对库更为友好，如更好的 ESM 产物支持。 社区上也有很多的工具在 Rollup 基础上进行了一定的封装，提供了对应用打包更友好的支持，如 vite 和 wmr, 目前 Rspack 比 Rollup 有更好的生产环境构建性能。
 
+### 和 Parcel 的区别
+
+Rspack 的整体架构上和 Parcel有很多共同之处，都把 css asset 视为一等公民，都支持了基于 filter 的 transformer, 但是 Parcel 更加注重开箱即用，而 Rspack更加注重为上层框架提供灵活的配置, Parcel 开创性的设计的 Unified Graph、HTML作为一等公民等特性，Rspack 在未来也会进行支持。
```
Comment (chenjiahan) [inline website/docs/zh/guide/start/introduction.mdx:130]:
```suggestion
Rspack 的整体架构与 Parcel 有很多共同之处。例如都将 CSS 资源视为一等公民，都支持基于 filter 的 transformer。然而，Parcel 更加注重开箱即用的体验，而 Rspack 更加注重为上层框架提供灵活的配置。Parcel 开创性地设计了 Unified Graph 和使 HTML 成为一等公民的特性。Rspack 也计划在未来支持这些特性。
```

**Comment by hardfist** [comment]
cc @devongovett

File: website/docs/en/guide/start/introduction.mdx [hunk 2]
```diff
@@ -111,6 +111,10 @@ Today Rspack is primarily tested using a subset of webpack's test cases. In the
 
 Rspack and Rollup are both bundling tools, but they focus on different areas. Rollup is more suitable for bundling libraries, while Rspack is more suitable for bundling applications. Therefore, Rspack has optimized many features for bundling applications, such as HMR and Bundle splitting. Rollup produces ESM outputs that are more friendly to libraries than Rspack. There are also many tools in the community that encapsulate Rollup to some extent and provide more friendly support for bundling applications, such as vite and wmr. Currently, Rspack has better production build performance than rollup.
 
+## Differences from Parcel
+
+The overall architecture of Rspack has a lot in common with Parcel, as both regard CSS and Assets as first-class citizens and both support filter-based transformers. However, Parcel focuses more on out-of-the-box usability, while Rspack focuses more on providing flexible configurations for upper-level frameworks and tools. The innovative features of Parcel like Unified Graph and treating HTML as first-class citizens are features that Rspack will support in the future."
```
Comment (chenjiahan) [inline website/docs/en/guide/start/introduction.mdx:116]:
```suggestion
The overall architecture of Rspack shares many similarities with Parcel. For example, both treat CSS assets as first-class citizens and both support filter-based transformers. However, Parcel focuses more on out-of-the-box usability, while Rspack focuses more on providing flexible configuration for upper-level frameworks and tools. Parcel innovatively designed features like the Unified Graph and making HTML a first-class citizen. Rspack also plans to support these features in the future.
```

File: website/docs/zh/guide/start/introduction.mdx [hunk 3]
```diff
@@ -125,6 +125,10 @@ Rspack 和 turbopack 都是基于 Rust 实现的 bundler，且都发挥了 Rust
 
 Rspack 和 Rollup 虽然都是打包工具，但是两者的侧重领域不同，Rollup 更适合用于打包库，而 Rspack 适合用于打包应用。因此 Rspack 对打包应用进行了很多优化，如 HMR、Bundle splitting 等功能，而 Rollup 则比 Rspack 的编译产物对库更为友好，如更好的 ESM 产物支持。 社区上也有很多的工具在 Rollup 基础上进行了一定的封装，提供了对应用打包更友好的支持，如 vite 和 wmr, 目前 Rspack 比 Rollup 有更好的生产环境构建性能。
 
+### 和 Parcel 的区别
+
+Rspack 的整体架构上和 [Parcel](https://parceljs.org/) 有很多共同之处，都把 css asset 视为一等公民，都支持了基于 filter 的 transformer, 但是 Parcel 更加注重开箱即用，而 Rspack更加注重为上层框架提供灵活的配置, Parcel 开创性的设计的 Unified Graph、HTML作为一等公民等特性，Rspack 在未来也会进行支持。
```
Comment (chenjiahan) [inline website/docs/zh/guide/start/introduction.mdx:130]:
```suggestion
Rspack 的整体架构与 [Parcel](https://parceljs.org/) 有很多共同之处。例如都将 CSS 资源视为一等公民，都支持基于 filter 的 transformer。然而，Parcel 更加注重开箱即用的体验，而 Rspack 更加注重为上层框架提供灵活的配置。Parcel 开创性地设计了 Unified Graph 和使 HTML 成为一等公民的特性。Rspack 也计划在未来支持这些特性。
```

File: website/docs/en/guide/start/introduction.mdx [hunk 4]
```diff
@@ -111,6 +111,10 @@ Today Rspack is primarily tested using a subset of webpack's test cases. In the
 
 Rspack and Rollup are both bundling tools, but they focus on different areas. Rollup is more suitable for bundling libraries, while Rspack is more suitable for bundling applications. Therefore, Rspack has optimized many features for bundling applications, such as HMR and Bundle splitting. Rollup produces ESM outputs that are more friendly to libraries than Rspack. There are also many tools in the community that encapsulate Rollup to some extent and provide more friendly support for bundling applications, such as vite and wmr. Currently, Rspack has better production build performance than rollup.
 
+## Compared with Parcel
+
+The overall architecture of Rspack has a lot in common with [Parcel](https://parceljs.org/), as both regard CSS and Assets as first-class citizens and both support filter-based transformers. However, Parcel focuses more on out-of-the-box usability, while Rspack focuses more on providing flexible configurations for upper-level frameworks and tools. The innovative features of Parcel like Unified Graph and treating HTML as first-class citizens are features that Rspack will support in the future."
```
Comment (chenjiahan) [inline website/docs/en/guide/start/introduction.mdx:116]:
```suggestion
The overall architecture of Rspack shares many similarities with [Parcel](https://parceljs.org/). For example, both treat CSS assets as first-class citizens and both support filter-based transformers. However, Parcel focuses more on out-of-the-box usability, while Rspack focuses more on providing flexible configuration for upper-level frameworks and tools. Parcel innovatively designed features like the Unified Graph and making HTML a first-class citizen. Rspack also plans to support these features in the future.
```

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**